types/classnames/classnames-tests.ts

```diff
@@ -38,3 +38,6 @@ const className = cx('foo', ['bar'], { baz: true }); // => "abc def xyz"
 
 // falsey values are just ignored
 cx(null, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
+
+// true is just ignored
+cx(true || "foo");
```

Comment:
what's the point of `|| "foo"` here?

Reply:
This is a typical use case in my code.;`classNames(true)` is useless, but `classNames(constExprToBeTrue || className)` is the case what I want to use.